fedimint-core/src/encoding/mod.rs

```diff
@@ -341,95 +341,6 @@ macro_rules! impl_encode_decode_tuple {
     );
 }
 
-/// Specialized version of Encodable for bytes
-pub fn consensus_encode_bytes<W: std::io::Write>(
-    bytes: &[u8],
-    writer: &mut W,
-) -> Result<usize, Error> {
-    let mut len = 0;
-    len += (bytes.len() as u64).consensus_encode(writer)?;
-    writer.write_all(bytes)?;
-    len += bytes.len();
-    Ok(len)
-}
-
-/// Specialized version of Encodable for static byte arrays
-pub fn consensus_encode_bytes_static<const N: usize, W: std::io::Write>(
-    bytes: &[u8; N],
-    writer: &mut W,
-) -> Result<usize, Error> {
-    writer.write_all(bytes)?;
-    Ok(bytes.len())
-}
-
-struct ReadBytesFromFiniteReaderOpts {
-    len: usize,
-    chunk_size: usize,
-}
-
-/// Read `opts.len` bytes from reader, where `opts.len` could potentially be
```

Comment:
Note for reviewer: I removed the mention of potential maliciousness since it isn't relevant here because we're always decoding data that we previously encoded.

Reply:
Removing unused code :+1: